public/javascripts/create-password.js

```diff
@@ -0,0 +1,75 @@
+const continueEvent = document.querySelector("#continue-button");
+const passwordFields = document.querySelector("#password-fields");
+const formGroup = document.querySelector(".govuk-form-group");
+const passwordInput = document.querySelector("#password");
+const confirmPasswordInput = document.querySelector("#confirm-password");
+const errorTextSection = document.querySelector("#error");
```

Comment:
nit: Apparently it's good to declare variable close to where they are used to increase readability (and cohesion) of code